### PR TITLE
[Edit] HTML Attributes: Linking to id #1603

### PR DESCRIPTION
**Pull request by muhiddinkurbonov**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/attributes/terms/id/id.md [hunk 1]
```diff
@@ -21,6 +21,28 @@ The HTML global **`id`** attribute is used to specify a unique identifier for an
...
+
+Specific and unique `id` attributes can be assigned to elements to differentiate them, and `id` values can be referenced in CSS and JavaScript to manipulate, format, and perform specific instructions on a single element alone, without affecting others of the same type.
+
+Valid `id` attribute values begin with a letter and must comprise only letters (`a`-`Z`), digits (`0`-`9`), hyphens (`-`), underscores (`_`), and periods (`.`).
+
+```html
+<h1 id="th1s_is.v4lid">Hello World</h1>
+```
+
+**Note:** While `th1s_is.v4lid` is indeed a valid `id` value, it's best to use a consistent naming scheme such as camel case (`thisIsValid`), dash case (`this-is-valid`), or snake case (`this_is_valid`).
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/id/id.md:34]:
```suggestion
> **Note:** While `th1s_is.v4lid` is indeed a valid `id` value, it's best to use a consistent naming scheme such as camel case (`thisIsValid`), dash case (`this-is-valid`), or snake case (`this_is_valid`).
```
This is bringing the old entry in line with the current style for "notes"

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@muhiddinkurbonov, I did a first review and had one comment to update the style of the old entry.

File: content/html/concepts/attributes/terms/id/id.md (hunk 1 above)
Comment (muhiddinkurbonov) [reply content/html/concepts/attributes/terms/id/id.md]: added > sign to fix the formatting.

**Review by SSwiniarski — APPROVED** [review]
@muhiddinkurbonov Looks good for a second review.

**Review by KTom101 — APPROVED** [review]
@muhiddinkurbonov I've added some minor changes (including removing some extra lines) and ran `npm run format:all` to fix the failing checks in commit [7fbde9a](https://github.com/Codecademy/docs/pull/1773/commits/7fbde9ad51e30582a759bd58690e7f25e0df24a2). It looks good to merge 👍

**Comment by KTom101** [comment]
@muhiddinkurbonov Thank you for your contributions. The edits are live and you're listed as a contributor for each: [<output>](https://www.codecademy.com/resources/docs/html/elements/output),
[Attributes](https://www.codecademy.com/resources/docs/html/attributes),[id](https://www.codecademy.com/resources/docs/html/attributes/id)

image

cc @yangc95

**Pull request merged**